# Define the goal
float64 yaw  # Specify current position
---
# Define the result
bool ok
---
# Define a feedback message
float64 currentYaw
